README.md

```diff
@@ -32,8 +32,11 @@ We want to create website with Django, that will be useful to as many as three d
 5. Install [pre-commit](https://pre-commit.com/)
 6. In terminal go to repository folder and enter and `pre-commit install`
 7. Open IDE (preferably PyCharm) and with `Open folder` option open repository folder as project in your IDE
+8. Install [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 
 ### Server running
 
 1. Open venv (if you use PyCharm just open terminal in GwardiaHub project)
-2. Enter `python manage.py runserver`
+2. Enter `docker compose up` (it may not work the first time, quit it with Ctrl+C and try again)
```

Comment:
It doesn't work
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/dd3ad404-1591-4847-89e7-ea01ebd5c88e)

Reply:
https://docs.docker.com/reference/cli/docker/compose/up/
Old versions used `docker-compose up`, but now it is merged into main `docker` command

Reply:
> It doesn't work ![image](https://private-user-images.githubusercontent.com/88268063/321709005-dd3ad404-1591-4847-89e7-ea01ebd5c88e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI4NTQwMzMsIm5iZiI6MTcxMjg1MzczMywicGF0aCI6Ii84ODI2ODA2My8zMjE3MDkwMDUtZGQzYWQ0MDQtMTU5MS00ODQ3LTg5ZTctZWEwMWViZDVjODhlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDExVDE2NDIxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjNjczNTAwZDNlMGNkMzc1NTFjOGUwNmZiYWFhYzkxZDBhMzBlYzg1MGZlNjllOTBiNjQ0NWNmYTEyYTI2ZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YIpt6GBAyjucnAtVqNJ9f7Eh-NsvrbRq8twGaphUFu8)

Yeah, that's an issue with your install